## **Задача 1.**


Продакт на главной mail.ru решил протестировать в рекомендательной ленте контента вместо карточек со статьями видеоплеер с короткими видео. Нынешний таймспент на юзера в день в среднем равен 25 минут, а стандартная ошибка (SD) равна 156. Мы предполагаем, что в новой версии таймспент на юзера в день изменится на 10%. Средний трафик 20000 человек в день. Посчитайте сколько дней необходимо держать эксперимент при alpha = 5% и beta = 20% .

Посчитаем ожидаемый таймспент на юзера в день для новый версии. 


25 + 10% = 27.7


SD = 156


Используя калькудятор с сайта https://www.statulator.com/SampleSize/ss2M.html посчитаем необходимую выборку. 


Получилось: 77858 на каждую группу. 


Разделим размер выборки на средний трафик и получим, сколько дней необходимо держать экспиремент. 

In [1]:
77858*2/20000

7.7858

В итоге получаем почти 8 дней эксперимента (7.8 дней)

## **Задача 2.**

Наша продуктовая команда в ecommerce магазине планирует запустить тест, направленный на ускорение загрузки сайта. Одна из основных метрик bounce rate в GA = 40%. Мы предполагаем, что при оптимизации сайта она изменится минимум на 20%.Средний трафик 4000 человек в день. Посчитайте сколько нам нужно дней держать эксперимент при alpha = 5% и beta = 20%.

Использую калькулятор https://www.evanmiller.org/ab-testing/sample-size.html при данных baseline = 40%, detectable effect = 20%, относительный показатель, alpha = 5% и beta = 20%, получили выборку 592 пользователя на вариант. 

In [2]:
592*2/4000

0.296

Получили 0.296 дня (или меньше 1го дня) нужно проводить эксперимент. 